gemfiles/ruby_2.6_rack_1.gemfile.lock

```diff
@@ -189,4 +189,4 @@ DEPENDENCIES
   yard (~> 0.9)
 
 BUNDLED WITH
-   2.3.26
```

Comment:
Just thinking out loud here - but wondering if we should:

- attempt to ignore/reset files that just have the `BUNDLED WITH` version as the only update
- attempt to use the same bundler version in each gemfile to do the update?